src/enum/gendersEnum.ts

```diff
@@ -0,0 +1,4 @@
+export enum GENDERS {
+  m = 'm',
+  f = 'f',
+}
```

Comment:
We need to get rid of magical values, not create new ones)
There are a lot of genders in the modern world, so it is not enough to simply indicate `f` and `m`
```suggestion
export enum Genders {
  Male = 'm',
  Female = 'f',
}
```